### ISSUE TITLE
-F no longer reports (estimated)track size

### ISSUE THREAD

**Issue by arter97** [rg3/youtube-dl#8480]
After merge
https://github.com/rg3/youtube-dl/commit/86e284e028cef6616260e2da5d5a260b1ef76fa7
youtube-dl -F no longer reports (estimated)track size.

Previously, it always printed DASH track size at the end of each lines.

Expected :
```
python ~/youtube-dl/youtube_dl/__main__.py -F https://www.youtube.com/watch?v=0bAeJ5fJ1M0
[youtube] 0bAeJ5fJ1M0: Downloading webpage
[youtube] 0bAeJ5fJ1M0: Downloading video info webpage
[youtube] 0bAeJ5fJ1M0: Extracting video information
[youtube] 0bAeJ5fJ1M0: Downloading DASH manifest
[youtube] 0bAeJ5fJ1M0: Downloading DASH manifest
[info] Available formats for 0bAeJ5fJ1M0:
format code  extension  resolution note
249          webm       audio only DASH audio   55k , opus @ 50k, 935.46KiB
250          webm       audio only DASH audio   80k , opus @ 70k, 1.36MiB
140          m4a        audio only DASH audio  127k , m4a_dash container, mp4a.40.2@128k (44100Hz), 2.23MiB
171          webm       audio only DASH audio  130k , vorbis@128k (44100Hz), 2.14MiB
251          webm       audio only DASH audio  154k , opus @160k, 2.61MiB
278          webm       256x144    DASH video  105k , webm container, vp9, 15fps, video only, 1.28MiB
160          mp4        256x144    DASH video  110k , avc1.42c00c, 15fps, video only, 1.92MiB
242          webm       426x240    DASH video  220k , vp9, 30fps, video only, 1.53MiB
133          mp4        426x240    DASH video  249k , avc1.4d4015, 30fps, video only, 4.30MiB
243          webm       640x360    DASH video  389k , vp9, 30fps, video only, 2.90MiB
134          mp4        640x360    DASH video  603k , avc1.4d401e, 30fps, video only, 4.19MiB
244          webm       854x480    DASH video  718k , vp9, 30fps, video only, 5.02MiB
135          mp4        854x480    DASH video 1102k , avc1.4d401f, 30fps, video only, 8.68MiB
247          webm       1280x720   DASH video 1424k , vp9, 30fps, video only, 10.12MiB
136          mp4        1280x720   DASH video 2201k , avc1.4d401f, 30fps, video only, 16.72MiB
248          webm       1920x1080  DASH video 2486k , vp9, 30fps, video only, 18.60MiB
137          mp4        1920x1080  DASH video 4133k , avc1.640028, 30fps, video only, 32.18MiB
271          webm       2560x1440  DASH video 8354k , vp9, 30fps, video only, 55.40MiB
264          mp4        2560x1440  DASH video 9921k , avc1.640032, 30fps, video only, 78.38MiB
313          webm       3840x2160  DASH video 16588k , vp9, 30fps, video only, 161.28MiB
266          mp4        3840x2160  DASH video 22112k , avc1.640033, 30fps, video only, 188.62MiB
17           3gp        176x144    small , mp4v.20.3,  mp4a.40.2@ 24k
36           3gp        320x240    small , mp4v.20.3,  mp4a.40.2@ 32k
5            flv        400x240    small , h263, mp3  @ 64k
43           webm       640x360    medium , vp8.0,  vorbis@128k
18           mp4        640x360    medium , avc1.42001E,  mp4a.40.2@ 96k
22           mp4        1280x720   hd720 , avc1.64001F,  mp4a.40.2@192k (best)
```

After that merge :
```
python ~/youtube-dl/youtube_dl/__main__.py -F https://www.youtube.com/watch?v=0bAeJ5fJ1M0
[youtube] 0bAeJ5fJ1M0: Downloading webpage
[youtube] 0bAeJ5fJ1M0: Downloading video info webpage
[youtube] 0bAeJ5fJ1M0: Extracting video information
[youtube] 0bAeJ5fJ1M0: Downloading MPD manifest
[youtube] 0bAeJ5fJ1M0: Downloading MPD manifest
[info] Available formats for 0bAeJ5fJ1M0:
format code  extension  resolution note
249          webm       audio only DASH audio   55k , opus @ 50k, 935.46KiB
250          webm       audio only DASH audio   80k , opus @ 70k, 1.36MiB
140          m4a        audio only DASH audio  127k , m4a_dash container, mp4a.40.2@128k (44100Hz)
171          webm       audio only DASH audio  130k , vorbis@128k (44100Hz)
251          webm       audio only DASH audio  154k , opus @160k, 2.61MiB
278          webm       256x144    DASH video  105k , webm container, vp9, 15fps, video only
160          mp4        256x144    DASH video  110k , avc1.42c00c, 15fps, video only
242          webm       426x240    DASH video  220k , vp9, 30fps, video only
133          mp4        426x240    DASH video  249k , avc1.4d4015, 30fps, video only
243          webm       640x360    DASH video  389k , vp9, 30fps, video only
134          mp4        640x360    DASH video  603k , avc1.4d401e, 30fps, video only
244          webm       854x480    DASH video  718k , vp9, 30fps, video only
135          mp4        854x480    DASH video 1102k , avc1.4d401f, 30fps, video only
247          webm       1280x720   DASH video 1424k , vp9, 30fps, video only
136          mp4        1280x720   DASH video 2201k , avc1.4d401f, 30fps, video only
248          webm       1920x1080  DASH video 2486k , vp9, 30fps, video only
137          mp4        1920x1080  DASH video 4133k , avc1.640028, 30fps, video only
271          webm       2560x1440  DASH video 8354k , vp9, 30fps, video only
264          mp4        2560x1440  DASH video 9921k , avc1.640032, 30fps, video only
313          webm       3840x2160  DASH video 16588k , vp9, 30fps, video only
266          mp4        3840x2160  DASH video 22112k , avc1.640033, 30fps, video only
17           3gp        176x144    small , mp4v.20.3,  mp4a.40.2@ 24k
36           3gp        320x240    small , mp4v.20.3,  mp4a.40.2@ 32k
5            flv        400x240    small , h263, mp3  @ 64k
43           webm       640x360    medium , vp8.0,  vorbis@128k
18           mp4        640x360    medium , avc1.42001E,  mp4a.40.2@ 96k
22           mp4        1280x720   hd720 , avc1.64001F,  mp4a.40.2@192k (best)
```

**Comment by remitamine** [comment]
i removed it because the filesize is not part of the MPEG-DASH standard it is an extention specific to youtube, other sites didn't use it(Facebook,Vevo,BBC and Kaltura).

**Comment by dstftw** [comment]
It should be extracted in case of Youtube anyway.

**Comment by arter97** [comment]
Can you workaround it by making it YouTube-specific(via extractor or something)?
I'm using a little wrapper on top of youtube-dl and this change completely breaks the percentage calculator, which needs some variables set with "-F" output to correctly calculate percentage based on audio/video track sizes.

**Comment by remitamine** [comment]
there are other extenstions and profiles. should we code specific to some sites in a generic method?

**Comment by dstftw** [comment]
For now we can just hardcode all extensions to the generic method like we do for some smil metadata.

**Comment by remitamine** [comment]
will be fixed in the next version.

**Comment by dstftw** [comment]
Fixed in youtube-dl 2016.02.09.1.
